token/users.go

```diff
@@ -0,0 +1,66 @@
+package token
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path/filepath"
+
+	"github.com/fabric8-services/fabric8-tenant/configuration"
+	"github.com/pkg/errors"
+)
+
+type userData struct {
+	Data struct {
+		Attributes struct {
+			Cluster string `json:"cluster,omitempty"`
+		}
+	}
+}
+
+type UserProfileService interface {
+	GetUserCluster(userID string) (string, error)
+}
+
+type UserController struct {
+	Config                     *configuration.Data
+	serviceAccountTokenService ServiceAccountTokenService
```

Comment:
The Users API is open and doesn't really need a service account token :)

Reply:
removed!